packages/wonder-blocks-dropdown/components/check-icons.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+/**
+ * Check and checkbox components used by option item.
+ */
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Color, {mix} from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+
+import type {IconAsset} from "@khanacademy/wonder-blocks-icon";
+
+const {
+    blue,
+    white,
+    offBlack,
+    offBlack16,
+    offBlack32,
+    offBlack50,
+    offWhite,
+} = Color;
+
+type CheckProps = {|
+    disabled: boolean,
+    selected: boolean,
+    pressed: boolean,
+    hovered: boolean,
+    focused: boolean,
+|};
+
+export const Check = (props: CheckProps) => {
```

Comment:
🛑 Hmm - I think I'd prefer having these be split into two files (we've been good about keeping 1 React component per file, exported using `export default class ...`). It doesn't look like there is very much shared between these two components, save for `CheckProps` and I think it's fine to duplicate that.